Submission1

```diff
@@ -0,0 +1,124 @@
+/* ----------------------------------------------------------------------------
+ * Copyright &copy; 2015 Jaemin Jin <jaemin_95@csu.fullerton.edu>
+ * Released under the [MIT License] (http://opensource.org/licenses/MIT)
+ * ------------------------------------------------------------------------- */
+ 
+/**
+ * Write a program that a number converts into a Roman Numeral.
```

Comment:
small english correction: `Write a program that converts a number into a Roman Numeral.`